### ISSUE TITLE
App-VOJournal-v0.4.7 fails tests on MSWin32

### ISSUE THREAD

**Issue by matwei** [matwei-perl/App-VOJournal#2]
Output from 'C:\Strawberry\perl\bin\perl.exe ./Build test':

```
# Testing App::VOJournal v0.4.7, Perl 5.032001, C:\Strawberry\perl\bin\perl.exe
t\00-load.t .............. ok
# Testing App::VOJournal::VOTL v0.4.7, Perl 5.032001, C:\Strawberry\perl\bin\perl.exe
t\VOTL.t ................. ok
t\boilerplate.t .......... ok
t\bug-107065.t ........... skipped: Script is not usable on MSWin32
insecure cwd in find(depth) at C:/Strawberry/perl/lib/File/Find.pm line 278.
t\find_find_last_file.t .. 
Dubious, test returned 25 (wstat 6400, 0x1900)
No subtests run 
t\manifest.t ............. skipped: Author tests not required for installation
t\pod-coverage.t ......... ok
t\pod.t .................. ok

Test Summary Report
-------------------
t\boilerplate.t        (Wstat: 0 Tests: 3 Failed: 0)
  TODO passed:   1-3
t\find_find_last_file.t (Wstat: 6400 (exited 25) Tests: 0 Failed: 0)
  Non-zero exit status: 25
  Parse errors: No plan found in TAP output
Files=8, Tests=24, 16 wallclock secs ( 0.12 usr +  0.03 sys =  0.16 CPU)
Result: FAIL
Failed 1/8 test programs. 0/24 subtests failed.
```
See for instance http://www.cpantesters.org/cpan/report/66d69970-6d3c-1014-b802-b35b9d98cdca

**Comment by matwei** [comment]
The code in questions is in *Find.pm*, in subroutine `_find_opt`:

```
 274         unless ( $no_chdir ) {
 275             if ( ($check_t_cwd) && (($untaint) && (is_tainted($cwd) )) ) {
 276                 ( $cwd_untainted ) = $cwd =~ m|$untaint_pat|;
 277                 unless (defined $cwd_untainted) {
 278                     die "insecure cwd in find(depth)";
 279                 }
 280                 $check_t_cwd = 0;
 281             }
 282             unless (chdir $cwd_untainted) {
 283                 die "Can't cd to $cwd: $!\n";
 284             }
 285         }
```
It looks like the option `no_chdir` can help, especially since we don't need to change the directory.
The man page for File::Find says:

>        "no_chdir"
>           Does not "chdir()" to each directory as it recurses. The "wanted()"
>           function will need to be aware of this, of course. In this case, $_
>           will be the same as $File::Find::name.

**Comment by matwei** [comment]
Fixed in v0.4.8
